ci/run_envoy_docker_windows_large_disk.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I think we can get rid of this file, https://github.com/envoyproxy/envoy/pull/13222 should give us the ability to pass an env var to modify the options to Docker